docs/proxy-config.md

```diff
@@ -105,3 +106,20 @@ OPTIONS
                                   'production' (default: sandbox)
     -o --output=<value>           Output format. One of: json|yaml
 ```
+
+### Deploy
+
+```shell
+NAME
+    deploy - Promotes the APIcast configuration to the Staging Environment
+
+USAGE
+    3scale proxy-config deploy <remote> <service>
+
+DESCRIPTION
+    Promotes the APIcast configuration to the Staging Environment (Production
+    Environment in case of Service Mesh).
```

Comment:
What's the particularity of service mesh?

Reply:
Copied from 3scale API docs. TBH I do not know. Maybe in ServiceMesh there is no staging env